tests/topotests/Dockerfile

```diff
@@ -53,7 +53,16 @@ RUN export DEBIAN_FRONTEND=noninteractive \
     && wget -qO- https://deb.frrouting.org/frr/keys.asc | apt-key add - \
     && echo "deb https://deb.frrouting.org/frr bionic frr-stable" > /etc/apt/sources.list.d/frr.list \
     && apt-get update \
-    && apt-get install -y libyang-dev \
+    && apt-get install -y \
```

Comment:
This change should go into a separate commit.

Reply:
ok